service/key/key_test.go

```diff
@@ -0,0 +1,86 @@
+package key
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+)
+
+// Test_Key_CAPath tests the CAPath function.
+func Test_Key_CAPath(t *testing.T) {
+	tests := []struct {
+		groupName    string
+		expectedPath string
+	}{
+		{
+			groupName:    "xa5ly",
+			expectedPath: "/certs/xa5ly/ca.pem",
+		},
+	}
```

Comment:
Nitpicking. I know there are cases where multiple test cases are not really possible or necessary but in general I think when you write table driven tests and you want to test the computation of information then you should at least have two or three cases.

Reply:
I disagree, slightly. It makes sense to me to write a table driven test, even if there's just one test case at the moment. Makes it easier to add more in the future.

Reply:
I mean just add another test with another ID to make sure there is no hard coded one.

Reply:
Done